src/SFA.DAS.EmployerIncentives.Commands/ApprenticeshipIncentive/SendEmploymentCheckRequests/SendEmploymentCheckRequestsCommandHandler.cs

```diff
@@ -1,25 +1,36 @@
 ﻿using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
 using SFA.DAS.EmployerIncentives.Abstractions.Commands;
 using SFA.DAS.EmployerIncentives.Commands.Persistence;
 using SFA.DAS.EmployerIncentives.Commands.Services.EmploymentCheckApi;
 using SFA.DAS.EmployerIncentives.Commands.Types.ApprenticeshipIncentive;
+using SFA.DAS.EmployerIncentives.Infrastructure.Configuration;
 
 namespace SFA.DAS.EmployerIncentives.Commands.ApprenticeshipIncentive.SendEmploymentCheckRequests
 {
     public class SendEmploymentCheckRequestsCommandHandler : ICommandHandler<SendEmploymentCheckRequestsCommand>
     {
         private readonly IApprenticeshipIncentiveDomainRepository _domainRepository;
         private readonly IEmploymentCheckService _employmentCheckService;
+        private readonly ApplicationSettings _applicationSettings;
 
-        public SendEmploymentCheckRequestsCommandHandler(IApprenticeshipIncentiveDomainRepository domainRepository, IEmploymentCheckService employmentCheckService)
+        public SendEmploymentCheckRequestsCommandHandler(IApprenticeshipIncentiveDomainRepository domainRepository, 
+                                                         IEmploymentCheckService employmentCheckService,
+                                                         IOptions<ApplicationSettings> applicationSettings)
         {
             _domainRepository = domainRepository;
             _employmentCheckService = employmentCheckService;
+            _applicationSettings = applicationSettings.Value;
         }
 
         public async Task Handle(SendEmploymentCheckRequestsCommand command, CancellationToken cancellationToken = default)
         {
+            if (!_applicationSettings.EmploymentCheckEnabled)
```

Comment:
Please implement this functionality as a decorator of the CommandHandler so it keeps the separation of concerns and can be easily tested and removed.

i.e

```
public class SendEmploymentCheckRequestsCommandHandlerWithEmploymentCheckToggle : ICommandHandler<SendEmploymentCheckRequestsCommand>

    {
        private readonly ICommandHandler<SendEmploymentCheckRequestsCommand> _dispatcher;
		private readonly ApplicationSettings _applicationSettings;

        public SendEmploymentCheckRequestsCommandHandlerWithEmploymentCheckToggle (
            ICommandHandler<SendEmploymentCheckRequestsCommand> dispatcher,
			 IOptions<ApplicationSettings> applicationSettings)
        {
            _dispatcher = dispatcher;  
_applicationSettings = applicationSettings;			
        }

        public async Task Handle(SendEmploymentCheckRequestsCommand command, CancellationToken cancellationToken = default)
        {
			// do check logic here
            await _dispatcher.Handle(command, cancellationToken);
        }
    }
```
    
and then register the decorator in the AddCommandHandlerDecorators method e.g.

```
serviceCollection.Decorate(typeof(ICommandHandler<SendEmploymentCheckRequestsCommand>), typeof(SendEmploymentCheckRequestsCommandHandlerWithEmploymentCheckToggle ));
```
after the common decorators.